### PR TITLE
#238 check bluetooth gps service ios

### PR DESCRIPTION
**Pull request by jannikbmc**

### CODE REVIEW

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/Services/CheckBluetoothAndLocationService.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    {
+                        Console.WriteLine(e.Message);
+                    }
+
+                    await Task.Delay(checkInterval);
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine(e.ToString());
+                }
```
Comment (Chippelius) [inline Implementation/FindMyBLEDevice/FindMyBLEDevice/Services/CheckBluetoothAndLocationService.cs:86]: I don't like the two directly nested try-catch-clauses. I would suggest to simply remove the inner one. 
You could move the await Task.Delay() to the top, but then the service would never get to actually check for permissions if for some reason the delay always fails / gets interrupted. I would rather have it the other way round, so the delay doesn't happen if something above it fails, the loop immediately restarts and tries again.

**Review by Chippelius — CHANGES_REQUESTED** [review]

File: Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/StrengthViewModel.cs [hunk 2]
```diff
@@ -138,8 +138,6 @@ private void InitializeCircleSizes()
 
         public async Task OnAppearing()
```
[the remark this answers is not in the document]
Comment (Wandinad) [reply Implementation/FindMyBLEDevice/FindMyBLEDevice/ViewModels/StrengthViewModel.cs]: Fixed

**Review by Wandinad — APPROVED** [review]

**Comment by Interperle** [comment]
works on iOS

**Review by Interperle — APPROVED** [review]

**Review by Chippelius — APPROVED** [review]
I approve now. 
However, there are merge conflicts now, please resolve them.

**Pull request merged**